app/models/staypuft/deployment/cinder_service.rb

```diff
@@ -134,6 +202,14 @@ def lvm_ptable
       end
     end
 
+    %w{hostname login password server_port storage_familie transport_type storage_protocol
```

Comment:
Any update on this? The goal is to get this into Staypuft ASAP and then on to OSP 6.